src/SFA.DAS.Authorization.Features/AuthorizationHandler.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace SFA.DAS.Authorization.Features
+{
+    public class AuthorizationHandler : IAuthorizationHandler
+    {
+        public string Prefix => "Feature.";
+        
+        private readonly IFeatureTogglesService<FeatureToggle> _featureTogglesService;
+        private readonly ILogger _logger;
+
+        public AuthorizationHandler(IFeatureTogglesService<FeatureToggle> featureTogglesService, ILogger<AuthorizationHandler> logger)
+        {
+            _featureTogglesService = featureTogglesService;
+            _logger = logger;
+        }
+
+        public Task<AuthorizationResult> GetAuthorizationResult(IReadOnlyCollection<string> options, IAuthorizationContext authorizationContext)
+        {
+            var authorizationResult = new AuthorizationResult();
+
+            if (options.Any())
```

Comment:
General obvs - .Any() and .Count() are used in places where .Length or .Count properties could be used instead. The previous is more consistent but the latter is more performant. Personal preference - just an obvs.

Reply:
Agreed, fixed.